src/components/ColormapEditor.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import {
+  Button,
+  Card,
+  CardBody,
+  CardSubtitle,
+  CardTitle,
+  Collapse,
+  Dropdown,
+  DropdownItem,
+  DropdownMenu,
+  DropdownToggle,
+  Input,
+  InputGroup,
+  InputGroupAddon,
+  ListGroup,
+  ListGroupItem
+} from "reactstrap";
+
+const default_colormaps = ['AMIP',
```

Comment:
This all looks good to me! But I think the default_colormaps const should be moved to the constants.ts file, no? That's where we have the default GM and other stuff.